prisma-fmt/src/text_document_completion.rs

```diff
@@ -66,12 +97,27 @@ fn push_ast_completions(
                 completion_list.items.push(CompletionItem {
                     label: referential_action.as_str().to_owned(),
                     kind: Some(CompletionItemKind::ENUM),
-                    // what is the difference between detail and documentation?
                     detail: Some(referential_action.documentation().to_owned()),
                     ..Default::default()
                 });
             }
         }
+
+        ast::SchemaPosition::DataSource(_source_id, ast::SourcePosition::Source) => {
+            if connector.has_capability(psl::datamodel_connector::ConnectorCapability::MultiSchema)
+                && schemas_span.is_none()
+                && preview_features.contains(PreviewFeature::MultiSchema)
+            {
+                completion_list.items.push(CompletionItem {
+                    label: "schemas".to_owned(),
+                    insert_text: Some("schemas = []".to_owned()),
```

Comment:
```suggestion
                    insert_text: Some("schemas = [$0]".to_owned()),
```
`$0` is the position of the cursor after the completion

Reply:
Trying to get this to work locally, doesn't seem to be as simple as just adding `$0` 👍

Reply:
Sorted it, it required this: `insert_text_format: Some(InsertTextFormat::SNIPPET)` seems to default to `PLAIN_TEXT`

Reply:
Indeed!